src/index.ts

```diff
@@ -277,19 +280,14 @@ class Marian {
       throw new InvalidQuery();
     }
 
-    const query = new Query(rawQuery);
+    let filters = extractFacetFilters(parsedUrl.searchParams);
```

Comment:
(Super nit) `const` instead of `let` here?

Reply:
done!